docs/api-docs/in-depth-guides/rail-integration-guide.mdx

```diff
@@ -1,100 +1,154 @@
-| Rail Unloaded  | `container.transport.rail_unloaded` | Notification when the container is unloaded from a railcar. |
-| Full Out       | `container.transport.full_out`      | Update when the full container leaves the rail terminal. |
-| Empty In       | `container.transport.empty_in`      | Notification when the empty container is returned to the terminal. |
 | Estimated Rail Arrival | `container.estimated.rail_arrival` | (Coming soon) Estimated time of arrival for the container at the rail terminal. |
 
-Integrate these notifications by subscribing to the webhooks and handling the incoming data to update your systems. 
+Integrate these notifications by subscribing to the webhooks and handling the incoming data to update your systems.
+
+> Note: We do not have webhook notifications planned for Train Passing, Rail Interchange Received, or Rail Interchange Delivered.  However, you can still see them in the dashboard.
 
 #### Rail Container Attributes
-- **Pickup Facility**: Information about the inland destination, rail terminal, and FIRMS code.
-- **Rail ETA**: Estimated Time of Arrival at the destination terminal.
-- **Rail ATA**: Actual Time of Arrival at the destination terminal.
-- **Available for Pickup**: Status indicating when the container is available for pickup.
-- **Demurrage LFD**: Last Free Day for demurrage charges.
-- **Rail Terminal Holds**: Holds placed on the container at the rail terminal.
-- **Rail Terminal Fees**: Fees associated with the container at the rail terminal.
 
-### Integration Methods
+The following are new attributes that are specific to rail container tracking.
 
-There are two primary methods to integrate Terminal49’s rail tracking data programmatically: via API and DataSync.
+- **pod_rail_loaded_at**: Time when the container is loaded onto a railcar at the POD.
+- **pod_rail_departed_at**: Time when the container departs from the POD.
+- **inland_destination_eta_at**: Estimated Time of Arrival at the destination terminal.
+- **inland_destination_ata_at**: Actual Time of Arrival at the destination terminal.
+- **inland_destination_rail_unloaded_at**: Time when the container is unloaded from the rail terminal.
+- **pickup_facility_holds**: Holds placed on the container at the rail terminal.
+- **pickup_facility_fees**: Fees associated with the container at the rail terminal.
+- **pickup_facility_yard_location**: Yard location of the container at the rail terminal.
+- **pickup_facility_lfd_on**: Last Free Day for demurrage charges.
+- **pickup_rail_carrier_scac**: SCAC code of the rail carrier that picks up the container from the POD (this could be different than the rail carrier that delivers to the destination terminal). {/* I think this is a bad name.  We should rename it to pod_rail_carrier_scac -- which is actually what it's named internally, but we change it for the serializer! */}
+{/* Why is there not a rail carrier scac for the delivery destination in the serializer?  We should add */}
```

Comment:
I agree - add a linear issue

Reply:
Linear issue: https://linear.app/terminal49/issue/DATA-2768/add-destination-rail-carrier-scac-to-serializer